src/composables/useSelectedMapInfo.ts

```diff
@@ -0,0 +1,43 @@
+import { storeToRefs } from "pinia";
+import { useAppStore } from "../stores/appStore";
+
+export const useSelectedMapInfo = () => {
+    const { mapSettings, admin1Indicators, admin2Indicators, admin1Geojson, admin2Geojson } =
+        storeToRefs(useAppStore());
+
+    const selectedIndicators = computed(() => {
+        // get single dictionary of feature id to indicator values
```

Comment:
I really appreciate the explanatory comments

Reply:
these were already there by you guys actually just kept them :laughing: